src/components/TodoItem/TodoItem.tsx

```diff
@@ -113,7 +114,7 @@ const ButtonWrapper = styled.div`
   }
 `
 const StyledButton = styled.button`
-  font-size: 0.8rem;
+  font-size: ${({ theme }) => theme.fontSize.small};
   padding: 2px 4px;
   border-radius: 2px;
```

Comment:
만약 이유가 있으시다면 이 부분과 해당 컨벤션이 지켜지지 않은 코드들은 다음과 같이 변경하면 좋을 것 같습니다!

```suggestion
  padding: 2px 4px;
  border-radius: 2px;
  
  font-size: ${({ theme }) => theme.fontSize.small};
```